api/src/pcapi/alembic/versions/20211214T082915_d8d88115998e_add_business_unit_status_column.py

```diff
@@ -0,0 +1,29 @@
+"""add_business_unit_status_column
+"""
```

Comment:
(détail)

```suggestion
"""Add BusinessUnit.status column"""
```

un peu plus clair (c'est affiché quand on lance `alembic history`)